Jenkinsfile

```diff
@@ -0,0 +1,12 @@
+node('docker') {
```

Comment:
Neglected to mention earlier, I think this should discard old builds automatically, e.g. 


```groovy
properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '32']]])
````